src/docs/forms/components/CheckBox/examples/01-SimpleCheckBox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import Form, {
+    CheckBox,
+    TextInput
+} from 'doj-react-adminlte/Form';
+
+export default class SimpleCheckBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            checkbox1: null,
```

Comment:
Never use null as initial state for properties that require boolean as a proptype. In this particular case, this will set the component to be uncontrolled.